docs/development_suite/clusters-management/vendors-runtime-services.md

```diff
@@ -1,11 +1,16 @@
 ---
 id: vendors-runtime-services
```

Comment:
L'ID e il nome del file lo lasciamo invariato?

Reply:
Changed in overview